fdk.go

```diff
@@ -114,6 +115,18 @@ type Context interface {
 
 	// FnID is Config()["FN_FN_ID"]
 	FnID() string
+
+	// AppName is Config()["FN_APP_ID"]
+	AppName() string
+
+	// FnName is Config()["FN_FN_Name"]
+	FnName() string
+
+	// ServiceName is Config()["FN_APP_ID"] + Config()["FN_FN_Name"]
+	ServiceName() string
```

Comment:
thanks for the changes, I think that ServiceName is part of the TracingContext as it has defined as part of that change. It could be confusing having a serviceName method here as users can expect different values. Inside the tracingContext, instead, it is more clear what it means.

Reply:
Yea I agree with Andrea, service name should be in the tracingContext as it is specific for tracing.